src/xdocs/config_annotation.xml

```diff
@@ -194,6 +202,28 @@ public String getNameIfPresent() { ... }
   &lt;property name=&quot;allowSamelineParameterizedAnnotation&quot; value=&quot;true&quot;/&gt;
 &lt;/module&gt;
         </source>
+        <p>
+          Example to allow only one and only parameterized annotation on the same line
+        </p>
+        <source>
+@SuppressWarnings("deprecation") DataLoader loader; // no violations
+@Mock DataLoader loader; // violation
+        </source>
+        <p>
+          Use the default configuration with tokens:
+        </p>
+        <source>
+&lt;module name=&quot;AnnotationLocation&quot;&gt;
+  &lt;property name=&quot;tokens&quot; value=&quot;METHOD_DEF&quot;/&gt;
+&lt;/module&gt;
+        </source>
+        <p>
+          Example to check default configuration with each method
+        </p>
+        <source>
+@Override public int hashCode() { ... } // no violations
+@SuppressWarnings("deprecation") public int foo() { ... } // violation
```

Comment:
Since we are only violating methods, we should add an example that is not a method that would have been violated on but now is ok.

Reply:
Agree with @rnveach. Please add a field or a class which has a parameterized annotation.

Reply:
Added a class with parameterized annotation.